tap_jira/streams.py

```diff
@@ -2386,6 +2388,23 @@ def get_records(self, context: dict | None) -> t.Iterable[dict[str, t.Any]]:
                 continue
             yield transformed_record
 
+    def validate_response(self, response: requests.Response) -> None:
+        """Validate the API response.
+
+        Allow for a 400 response if the board does not support sprints.
+        Do raise an error for other 400 responses.
+        """
+        try:
+            super().validate_response(response)
+        except FatalAPIError:
+            if (
+                response.status_code == HTTPStatus.BAD_REQUEST
+                and "The board does not support sprints"
+                in response.json().get("errorMessages", [])
+            ):
+                return
+            raise
```

Comment:
I think we can do something like

```suggestion
        if (
            response.status_code == HTTPStatus.BAD_REQUEST
            and "The board does not support sprints"
            in response.json().get("errorMessages", [])
        ):
            return
        super().validate_response(response)
```

and save us one try-except block.

Reply:
and also the FatalAPIError import, neat